src/MockedResponse.ts

```diff
@@ -10,4 +10,5 @@ export interface MockedResponse<TData = Record<string, any>> {
   // Homebound note: Added as part of making MockLink behavior more sane/less opaque.
   loading?: boolean;
   newData?: ResultFunction<FetchResult>;
+  requested?: number;
```

Comment:
Thoughts on calling this `requestCount` or something similar?

Reply:
@JoshuaScript maybe `requestedCount`? I like the past-tense of "requested" because when present-tense `request` I worry that it would read as "when creating this mock, I want this mock to be used 3 times" instead of the retrospective "this mock _was_ used 3 times".